tools/Environments/DevHome.Environments/Views/LandingPage.xaml

```diff
@@ -35,38 +35,39 @@
-                        Command="{x:Bind LaunchActionCommand}"
-                        x:Uid="LaunchButton"
-                        Style="{StaticResource CardBodySplitButtonStyle}">
-                        <SplitButton.Flyout>
-                            <customControls:CardFlyout ItemsViewModels="{x:Bind LaunchOperations, Mode=OneWay}"/>
-                        </SplitButton.Flyout>
-                    </SplitButton>
-
-                    <!-- A single button template is shown when the extension only supports launching from the launch button.-->
-                    <Button
-                        Visibility="{x:Bind ShouldShowSplitButton, Mode=OneWay, Converter={StaticResource NegatedBoolToVisibilityConverter}}"
-                        VerticalAlignment="Top"
-                        Command="{x:Bind LaunchActionCommand}"
-                        x:Uid="LaunchButton"
-                        Style="{StaticResource CardBodyLaunchButtonStyle}">
-                    </Button>
-                </StackPanel>
+            <DataTemplate x:Key="LaunchButton" x:DataType="vm:ComputeSystemViewModel">
+                <StackPanel
+                    Visibility="{x:Bind ShouldShowLaunchOperation, Mode=OneWay}">
+                    <!-- The split button with a menu flyout is only shown when there are other operations the extension supports other than launching -->
+                    <SplitButton
+                        Visibility="{x:Bind ShouldShowSplitButton, Mode=OneWay}"
+                        VerticalAlignment="Top"
+                        Command="{x:Bind LaunchActionCommand}"
+                        x:Uid="LaunchButton"
+                        Style="{StaticResource CardBodySplitButtonStyle}">
+                        <SplitButton.Flyout>
+                            <customControls:CardFlyout ItemsViewModels="{x:Bind LaunchOperations, Mode=OneWay}"/>
+                        </SplitButton.Flyout>
+                    </SplitButton>
+
+                    <!-- A single button template is shown when the extension only supports launching from the launch button.-->
+                    <Button
+                        Visibility="{x:Bind ShouldShowSplitButton, Mode=OneWay, Converter={StaticResource NegatedBoolToVisibilityConverter}}"
+                        VerticalAlignment="Top"
+                        Command="{x:Bind LaunchActionCommand}"
+                        x:Uid="LaunchButton"
+                        Style="{StaticResource CardBodyLaunchButtonStyle}">
+                    </Button>
+                </StackPanel>
             </DataTemplate>
 
             <!-- Three Dot Button template -->
             <DataTemplate x:Key="ThreeDotsButton" x:DataType="vm:ComputeSystemViewModel">
                 <Grid>
                     <Button
                         Style="{StaticResource HorizontalThreeDotsStyle}"
-                        AutomationProperties.Name="{x:Bind MoreOptionsButtonName}">

```

Comment:
Would it be possible to use `x:UID="MoreOptionsButton"` instead of explicitly setting properties here?
Same below

Reply:
That seems to work.

Was anyone able to test the "More options" tooltip for a Hyper-V environment? I tried to set one up on the branch build, but when I completed the setup process, no list item in the list of Hyper-V environments appeared under the "Hyper-V" heading indicating the environment was actually created.

<img width="836" alt="image" src="https://github.com/user-attachments/assets/8243dbeb-76cd-45d0-b2c3-25464d0b8b1f">

Reply:
I believe you can also verify or create a Hyper-V from the Hyper-V Manager, and Dev Home would reflect the change (after re-sync). CC: @bbonaby

Reply:
That worked for me. Thanks!